### DISCUSSION TITLE
how should I install this app

### DISCUSSION THREAD

**Discussion by nakul373** [libre-tube/LibreTube#5556] [General]
I am new to github. Please help me with installation of this app in my android device.

**Comment by BlackSparkz** [comment]
> I am new to github. Please help me with installation of this app in my android device.

https://github.com/libre-tube/LibreTube/releases/download/v0.21.1/app-arm64-v8a-release.apk

**Comment by brigazvi** [comment]
You can also use obtanium to update it automatically
